### PR TITLE
Core & react/typing fixes

### PR DESCRIPTION
**Pull request by dpinol**
fixes in core & react types

### CODE REVIEW

**Review by ericmarcos — APPROVED** [review]

File: packages/botonic-plugin-contentful/tests/contentful/performance.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        locale: 'es',
+      },
+      (cf: CommonFields) => cf.partitions && cf.partitions.includes('ES')
+    )
+    console.log(res.length)
+    console.log('hola')
+    searchM.end()
+    // expect(res.length).toBeGreaterThanOrEqual(1)
+    // expect(res.filter(res => res.common.name == 'POST_FAQ1')).toHaveLength(1)
+    // console.log(stringSummaryAll())
```
Comment (asastre) [inline packages/botonic-plugin-contentful/tests/contentful/performance.integration.test.ts:38]: remove `console.log` and commented lines?

**Review by asastre — COMMENTED** [review]
I have some doubts/observations before approving

File: packages/botonic-plugin-contentful/src/cms/decorators/cms-stats.ts [hunk 2]
```diff
@@ -0,0 +1,307 @@
+// /* eslint-disable @typescript-eslint/unbound-method */
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply packages/botonic-plugin-contentful/src/cms/decorators/cms-stats.ts:1]: removing it, we can recover it whenever it is necessary

File: packages/botonic-plugin-contentful/src/util/profiler.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+export const isBrowser = () => {
+  // @ts-ignore
+  return typeof IS_BROWSER !== 'undefined'
+    ? // eslint-disable-next-line no-undef
+      // @ts-ignore
+      IS_BROWSER
+    : typeof window !== 'undefined' &&
+        typeof window.document !== 'undefined' &&
+        !window.process
+}
```
[the remark this answers is not in the document]
Comment (vanbasten17) [reply packages/botonic-plugin-contentful/src/util/profiler.ts:16]: the `measurer` module is missing in the src code

**Comment by vanbasten17** [comment]
Closing, already included in: https://github.com/hubtype/botonic/pull/1649

**Pull request closed without merging**